tests/manifests/strimzi/test_kafka_topic.py

```diff
@@ -48,6 +48,7 @@ def test_topic_spec_validation():
 def test_strimzi_kafka_topic_from_topic(kafka_topic: KafkaTopic, mocker: MockerFixture):
     mock_config = MagicMock()
     mock_config.strimzi_topic.cluster_labels = ("bakdata.com/cluster", "my-cluster")
+    mock_config.strimzi_topic.namespace = None
```

Comment:
isn't that the default value?

Reply:
It fails due to mocker assigning an unexpected type, let me get it to be more concrete:

```python
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for ObjectMeta
E       namespace
E         Input should be a valid string [type=string_type, input_value=<MagicMock name='mock.str...espace' id='4596251488'>, input_type=MagicMock]
E           For further information visit https://errors.pydantic.dev/2.10/v/string_type
```

Reply:
alright, then it's fine